src/Notify/Client/NotificationClient.cs

```diff
@@ -102,11 +103,30 @@ public TemplateList GetAllTemplates(String templateType = "")
             return templateList;
         }
 
-        public SmsNotificationResponse SendSms(String mobileNumber, String templateId, Dictionary<String, dynamic> personalisation = null, String clientReference = null)
+        public ReceivedTextListResponse GetReceivedTexts(String olderThanId = "")
+        {
+            String finalUrl = String.Format(
+                "{0}{1}",
+                GET_RECEIVED_TEXTS_URL,
+                String.IsNullOrWhiteSpace(olderThanId) ? "" : "?older_than=" + olderThanId
+            );
+
+            String response = this.GET(finalUrl);
+
+            ReceivedTextListResponse receivedTexts = JsonConvert.DeserializeObject<ReceivedTextListResponse>(response);
```

Comment:
Should check for null or empty here and raise an exception?

Reply:
Likewise we don't do this anywhere else, and an empty response is acceptable as a service might not have any received text messages